content/webapp/pages/exhibition-guide.tsx

```diff
@@ -294,7 +294,11 @@ const Stops: FC<StopsProps> = ({ stops, type }) => {
             audioWithoutDescription?.url) ||
           (type === 'bsl' && bsl?.embedUrl);
         return hasContentOfDesiredType ? (
-          <Stop key={index} id={dasherizeShorten(title)}>
+          <Stop
+            key={index}
+            id={'apiToolbar'}
+            data-toolbar={dasherizeShorten(title)}
```

Comment:
So this now will be the anchor name, is that right?

Reply:
yes, that's right

Reply:
Sorry I edited the comment after 😄

Reply:
Yep, good shout. I've changed that now. Took me a minute to remember `data-toolbar-anchor` resolves to `toolbarAnchor` as an attribute

Reply:
Taking `el.dataset` from an id, or `getAttributes()` doesn’t like any more than one dash after `data-` so, when you select a data attribute with `dataset` or `getAttributes()` something like `data-a-long-name` becomes `aLongName`